app/helpers/button_helper.rb

```diff
@@ -3,6 +3,10 @@ def sign_up_button
     link_to 'Sign Up', sign_up_path(ref: 'success_stories'), class: 'button button--primary'
   end
 
+  def curriculum_button
+  	link_to 'Curriculum', curriculum_path(ref: 'success_stories'), class: 'button button--primary'
```

Comment:
Indentation is still off here, please take a look at the other methods in this file to see how it should be indented.